invoker.js

```diff
@@ -406,7 +406,8 @@ export function apply() {
   applyInvokerMixin(HTMLButtonElement);
 
   observeShadowRoots(HTMLElement, (shadow) => {
-    setupInvokeListeners(shadow);
+    // Make sure to not re-attach listeners here, because click events are composable and already pierce
+    // the shadow dom, so the click listener on the document is enough
```

Comment:
Okay this makes a bit more sense but doesn't this mean that the `click` events that are observed outside the shadow root will have a different target? Meaning the code won't be able to read buttons inside shadowroots for their command/commandfor attributes?

I think perhaps a better way to solve this is to add a WeakSet of "handled" events and check the WeakSet, returning early if the event is present.